src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -675,6 +675,11 @@
     "NAME": "Shake",
     "DESCRIPTION": "If enemy is Vulnerable, draw !M! cards. NL Apply !stsjorbsmod:UrMagic! Vulnerable. NL Exhaust."
   },
+  "stsjorbsmod:ShrapnelBloom": {
+    "NAME": "Shrapnel Bloom",
+    "DESCRIPTION": "stsjorbsmod:Exert. NL This turn, the next non-Legendary card you play is played !M! additional time, then exerts.",
```

Comment:
The term "exerts" at the end of the 2 descriptions should probably also be capitalized/colorized like a keyword (`stsjorbsmod:Exerts` here, `#yExerts` in the power description)

Reply:
"Exerts" was not a Keyword, added that and updated.